sycl/doc/GetStartedGuide.md

```diff
@@ -298,6 +299,14 @@ Enabling this flag requires following packages installed.
 Currently, this feature was tested and verified on Ubuntu 20.04
 environment.
 
+### Build DPC++ toolchain with support for runtime kernel fusion
+
+Support for the experimental SYCL extension for user-driven kernel fusion 
+at runtime is enabled by default. 
+
+To disable support for this feature, follow the instructions for the 
+Linux DPC++ toolchain, but add the `--disable-fusion` flag.
+
```

Comment:
Could you please add a note that this feature is not yet supported on the Windows platform?